File: Backend/NotdienstPlaner/Database/Model/Pharmacy.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Notdienstplaner.Database.Model;

public class Pharmacy
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    [Key]
    public Guid Id { get; set; }

    public string Name { get; set; }

    public string Email { get; set; }

    public string Phone { get; set; }

    public string Place { get; set; }

    public string Street { get; set; }

    public string Housenumber { get; set; }
    
    public string Postcode { get; set; }

}

File: Backend/model/Pharmacy.py
from .. import db


class Pharmacy(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    name = db.Column(db.String(80), unique=True, nullable=False)
    email = db.Column(db.String(80))
    phone = db.Column(db.String(80))
    place = db.Column(db.String(80))
    street = db.Column(db.String(80))
    houseNumber = db.Column(db.Integer)
    postcode = db.Column(db.Integer)